api/v1/endpoints/listings.py

```diff
@@ -89,4 +86,7 @@ def post(self):
             timings['s3_save'] = time.time() - local_finish
             os.remove(f'{destination}{filename}')
         log.info(timings)
-        return 'You uploaded a file'
+        db = dynamo.dynamo_conn
+        db_entry = db.add_listing(payload)
+        if db_entry == constants.DB_SUCCESS:
+            return {'message': constants.NEW_LISTING_SUCCESS}, 201
```

Comment:
its not a listing yet -- it's still a candidate at this step in the protocol's eyes. may not matter, but thought i'd mention it

Reply:
Fair point. Words matter and I think in this case it can lead to misunderstanding